engine/src/flutter/shell/platform/darwin/ios/framework/Source/ConnectionCollectionTest.swift

```diff
@@ -0,0 +1,20 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import InternalFlutterSwift
+import XCTest
+
+class ConnectionCollectionTest: XCTestCase {
+  func testAcquireAndRelease() {
+    let connections = ConnectionCollection()
+    let connection = connections.acquireConnection(channel: "foo")
```

Comment:
i'd also assert the value of connection here.

Reply:
I've added a test to verify that the result is a positive integer value since that's what we advertise. We don't provide any guarantees about the value of the ID in the API contract. There's no reason we need to return a series of integers starting at 1 and nothing would break if we changed this, so testing that we're a positive int is likely enough.